config/locales/en.yml

```diff
@@ -30,6 +30,14 @@
 # available at https://guides.rubyonrails.org/i18n.html.
 
 en:
+  settings_menu:
```

Comment:
Sorry for being so nitpicky but could we keep this file alphabetically sorted, so put `settings_menu` after `core`?

Reply:
Thanks a lot @metamoni